uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/qmldir

```diff
@@ -54,6 +54,7 @@ Scalebar 100.13 Scalebar.qml
 Scalebar 200.0 Scalebar.qml
 SearchView 100.13 SearchView.qml
 SearchView 200.0 SearchView.qml
+ServerTrustView 200.0 ServerTrustView.qml
```

Comment:
Forgot about this file... With the new Qt 6 QML version changes, I wonder if there's opportunity to simplify this whole story? Obviously not as part of this PR...

Reply:
Yeah good point. There are likely a few things we can clean up. I don't want to push things with this PR.